src/components/FlowRunTimeline.vue

```diff
@@ -49,16 +49,20 @@
           :is-running="isRunning"
           :format-date-fns="formatDateFns"
           :theme="theme"
-          :selected-node-id="selectedNode"
-          @click="selectNode"
+          :sub-node-labels="subFlowRunLabels"
+          :selected-node-id="selectedNode?.id"
+          :expanded-sub-nodes="expandedSubFlowRuns"
+          @selection="selectNode"
```

Comment:
I opted to make the selected state owned by the consumer. I could see it being appended to the URL so a link can be shared to a particular part, or maybe search functionality in the future results in a node selection etc. I might have an IOU for v-model here, I agree it might as well be, but unless I have another item sending me to update the graphs repo, I might do that in another PR so I can get this to staging.

Reply:
Yeah fair enough, no need to update the graphs repo, just commenting that's probably a pattern more in line with how you're using it

Reply:
Will update this next graphs update for sure.